src/components/Combobox/PCombobox.vue

```diff
@@ -147,11 +147,11 @@
   function displayValue(value: SelectModelValue): string {
     const option = getSelectOption(value)
 
-    if (option?.value) {
+    if (option) {
       return option.label
     }
 
-    if (!value) {
+    if (value === null) {
```

Comment:
Do we need to use a nullish check instead? `value === null || value === undefined`?

Reply:
That's possible but I don't think so. Based on the types
<img width="770" alt="image" src="https://user-images.githubusercontent.com/6200442/221995319-e0c7b132-2545-4f90-9585-d6909643e132.png">

Reply:
I tested that a modelValue of `undefined` works as expected. So think the types are correct.

Reply:
I appreciate the follow-up!